pkg/k8sutil/perbrokerconfig.go

```diff
@@ -0,0 +1,40 @@
+// Copyright © 2020 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package k8sutil
+
+// these configurations will not trigger rolling upgrade when updated
+var PerBrokerConfigs = []string{
```

Comment:
Not sure if the `k8sutil` package is the right place for this list as it is specific to Kafka brokers. I'd place it in an internal broker config package an make it private. Since this list should have no duplicates probable is better to store it in a `map[string]struct{}`

Reply:
Reconciling k8s resources are performed through `k8sutil/resources.go` 's `Reconcile` function. This function depends on the Kafka controller package, because this function sets the config in sync/out of sync states. This causes a circular dependency when I moved it into the kafka package. Where should it be put then?

Reply:
I'd move out the updating of the broker status when the config map changes into `kafka.go` where the config map change originates from.

Reply:
Moved the related parts to the kafka package.